en/03_Drawing_a_triangle/04_Swap_chain_recreation.md

```diff
@@ -18,23 +18,15 @@ void recreateSwapChain() {
 
     createSwapChain();
     createImageViews();
-    createRenderPass();
-    createGraphicsPipeline();
     createFramebuffers();
 }
 ```
 
 We first call `vkDeviceWaitIdle`, because just like in the last chapter, we
 shouldn't touch resources that may still be in use. Obviously, the first thing
 we'll have to do is recreate the swap chain itself. The image views need to be
```

Comment:
```suggestion
We first call `vkDeviceWaitIdle`, because just like in the last chapter, we
shouldn't touch resources that may still be in use. Obviously, we'll have
recreate the swap chain itself. The image views need to be
```
(just noticed the language here says 'first' twice, which is redundant and a bit odd)